proto/storage/image.proto

```diff
@@ -9,10 +9,14 @@ import "gogoproto/gogo.proto";
 
 package storage;
 
-// Next Tag: 18
+// Next Tag: 19
 message Image {
     string                          id                          = 4 [(gogoproto.moretags) = 'search:"Image Sha,store,hidden" sql:"pk"'];
     ImageName                       name                        = 1;
```

Comment:
Do we want to put `deprecated` on this field? Or not yet?

Reply:
I'd say not yet.